packages/api/src/test-runner/run-result-helpers.ts

```diff
@@ -11,16 +11,17 @@ export function determineHitLimitReached(hitCount: number | undefined, hitLimit:
   return;
 }
 
-export function toMutantRunResult(dryRunResult: DryRunResult): MutantRunResult {
+export function toMutantRunResult(dryRunResult: DryRunResult, reportAllKillers = false): MutantRunResult {
```

Comment:
I'm not a huge fan of this boolean parameter (I have repeated for years that every time someone creates a boolean parameter a puppy dies 🐶 ) but I didn't think creating a type of options would be necessary. 

open for suggestions

Reply:
Haha 😀

Well, ... I don't think the boolean is needed here. We can simply always filter and fill the array that way. Performance is not an issue, because if bail was active, the final test would always be the failing one 🤷‍♀️

🐶

Reply:
I added the boolean because if `toMutantRunResult ` starts returning an array it'd break some of the test from other test runners. For example [this test](https://github.com/stryker-mutator/stryker-js/blob/master/packages/mocha-runner/test/integration/sample-project-instrumented.it.spec.ts#L114)

So I think it is necessary while we implement this for the other tests runners.

What do you think?